models/models.reviews.js

```diff
@@ -37,3 +38,25 @@ exports.fetchCommentsByReviewId = (review_id) => {
       return res.rows;
     });
 };
+
+exports.updateReview = (review_id, inc_votes) => {
+  let countValue;
+  if (inc_votes < 0) {
+    countValue = format(
+      `UPDATE reviews SET votes = votes + ${inc_votes} WHERE review_id = ${review_id} RETURNING *;`
```

Comment:
These queries are exactly the same, which is suggesting that either your test passed the first time and you wrote the extra query any way or you didn't run the tests before writing this code 👀

Reply:
haha this has left me so confused, i thought id included a - instead of a plus and it passed fine but now im unsure why its passing at all with a + when im trying to decrement?! I don't think my maths is wrong (all though usually quite bad) and im a bit scared of this so for now I am going to leave it and have a think about it